### PR TITLE
chore: changelog for 473

### PR DESCRIPTION
**Pull request by Conni2461**
CC @clason

### CODE REVIEW

File: lua/telescope/utils.lua [hunk 1]
```diff
@@ -251,6 +251,10 @@ utils.diagnostics_to_tbl = function(opts)
   return items
 end
 
+utils.path_shorten = function(file)
```
Comment (Conni2461) [inline lua/telescope/utils.lua:254]: We should have kept this as shorthand, but make it work for the new interface

Comment (clason) [reply lua/telescope/utils.lua:254]: That's probably the biggest one, yes.

**Comment by clason** [comment]
Yes, that is very helpful, thank you!

**Comment by clason** [comment]
What about `path_tail`?

**Comment by Conni2461** [comment]
`path_tail` was always in utils.

**Pull request merged**